drivers/ieee802154/Kconfig.nrf5

```diff
@@ -42,4 +42,12 @@ config IEEE802154_NRF5_EXT_IRQ_MGMT
 	  the system. One example of external radio IRQ provider could be
 	  a radio arbiter used in dynamic multiprotocol applications.
 
+config IEEE802154_NRF5_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 16043574
```

Comment:
Then, this could be Zephyr Project OUI, if we had one, otherwise networking folks can suggest better value. I'm thinking to use same approach from USB_DEVICE_VID, for instance.

Reply:
I answered on this in above comment